src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/StackWalk/FrameHandling/FrameIterator.cs

```diff
@@ -158,7 +159,7 @@ public static string GetFrameName(Target target, TargetPointer frameIdentifier)
 
     public FrameType GetCurrentFrameType() => GetFrameType(target, CurrentFrame.Identifier);
 
-    private static FrameType GetFrameType(Target target, TargetPointer frameIdentifier)
+    public static FrameType GetFrameType(Target target, TargetPointer frameIdentifier)
```

Comment:
This doesn't need to be changed anymore
```suggestion
    private static FrameType GetFrameType(Target target, TargetPointer frameIdentifier)
```